frontend/packages/dev-console/src/components/topology/__tests__/topology-utils.spec.ts

```diff
@@ -220,4 +221,20 @@ describe('TopologyUtils ', () => {
     ]);
     expect((topologyTransformedData[keys[0]].data as WorkloadData).editUrl).toBe(mockGitURL);
   });
+
+  it('should return builder image icon for nodejs', () => {
+    const nodejsIcon = getImageForIconClass('icon-nodejs');
+    const { topologyTransformedData, keys } = getTranformedTopologyData(MockResources, [
+      'deploymentConfigs',
+    ]);
+    expect((topologyTransformedData[keys[0]].data as WorkloadData).builderImage).toBe(nodejsIcon);
+  });
+
+  it('should return builder image icon for nodejs', () => {
```

Comment:
nit: it would be good to use different description in `it` block for this one and above

Reply:
Sorry, that was because of copy paste job. Updated the description.